src/ol/AssertionError.js

```diff
@@ -61,6 +61,7 @@ const messages = {
   67: 'A layer can only be added to the map once. Use either `layer.setMap()` or `map.addLayer()`, not both',
   68: 'A VectorTile source can only be rendered if it has a projection compatible with the view projection',
   69: '`width` or `height` cannot be provided together with `scale`',
+  70: 'Cannot create polygon from empty extent',
```

Comment:
We have deprecated `ol/AssertionError`. Please throw an exception directly in `ol/geom/Polygon.js`.